src/background/RemotePontoon.ts

```diff
@@ -1,6 +1,6 @@
 import URI from 'urijs';
 
-import type { StorageContent } from '@commons/webExtensionsApi';
+import { StorageContent } from '@commons/webExtensionsApi';
```

Comment:
```suggestion
import type { StorageContent } from '@commons/webExtensionsApi';
```

This should be kept as type import only. ESLint will likely complain about that.